From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Thu, 12 Dec 2024 00:11:00 -0600
Subject: [PATCH] Solus: Make it easier to set version from ypkg

ypkg handles escaping quotes very poorly, so let's just do this so we can use a sed to set the version string

---
 clang/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clang/CMakeLists.txt b/clang/CMakeLists.txt
index c6496167d382..26d023c2aa0c 100644
--- a/clang/CMakeLists.txt
+++ b/clang/CMakeLists.txt
@@ -278,7 +278,7 @@ set(CLANG_SYSTEMZ_DEFAULT_ARCH "z10" CACHE STRING "SystemZ Default Arch")
 set(CLANG_VENDOR ${PACKAGE_VENDOR} CACHE STRING
   "Vendor-specific text for showing with version information.")
 
-set(CLANG_REPOSITORY_STRING "" CACHE STRING
+set(CLANG_REPOSITORY_STRING "@@SOLUS_VERSION_STRING@@" CACHE STRING
   "Vendor-specific text for showing the repository the source is taken from.")
 
 if(CLANG_REPOSITORY_STRING)
